txinput.go

```diff
@@ -37,20 +36,20 @@ func NewInputFromBytes(bytes []byte) (*Input, int, error) {
 
 // TotalInputSatoshis returns the total Satoshis inputted to the transaction.
 func (tx *Tx) TotalInputSatoshis() (total uint64) {
-	for _, in := range tx.Inputs {
+	for _, in := range tx.inputs {
 		total += in.PreviousTxSatoshis
 	}
 	return
 }
 
 func (tx *Tx) addInput(input *Input) {
-	tx.Inputs = append(tx.Inputs, input)
+	tx.inputs = append(tx.inputs, input)
 }
 
 // AddInputFromTx will add all outputs of given previous transaction
```

Comment:
we should specify that this is only for p2pkh, shouldn't we?

Reply:
Oh the comment? yeah I can update that